crates/cast/bin/args.rs

```diff
@@ -539,6 +539,16 @@ pub enum CastSubcommand {
         data: String,
     },
 
+    /// Decode custom error data.
+    #[command(visible_aliases = &["error-decode", "--error-decode", "erd"])]
+    DecodeError {
+        /// The error signature. If none provided then tries to decode from local cache or `https://api.openchain.xyz`.
+        #[arg(long, visible_alias = "error-sig")]
+        sig: Option<String>,
```

Comment:
can we make this optional for decode event too?

Reply:
yep, will follow up with a PR, have to figure out how to parse with topics